main.py

```diff
@@ -2,6 +2,27 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
```

Comment:
There is a typo in the actor's first name 'Kianu'. It should be 'Keanu'. This is corrected later in the code, but it's better to avoid typos initially.